### PR TITLE
Added Xray offline-update flags to support Xray's DBSync V3 flags

### PR DESCRIPTION
**Pull request by itraviv**
- [x] All [tests](https://github.com/jfrog/jfrog-cli#tests) passed. If this feature is not already covered by the tests, I added new tests.
- [x] All [static analysis checks](https://github.com/jfrog/jfrog-cli/actions/workflows/analysis.yml) passed.
- [x] This pull request is on the dev branch.
- [x] I used gofmt for formatting the code before submitting the pull request.
-----

### CODE REVIEW

**Comment by itraviv** [comment]
I have read the CLA Document and I hereby sign the CLA

File: utils/cliutils/commandsflags.go [hunk 1]
```diff
@@ -1151,6 +1153,14 @@ var flagsMap = map[string]cli.Flag{
 		Name:  target,
 		Usage: "[Default: ./] Path for downloaded update files.` `",
 	},
+	DBSyncV3: cli.BoolFlag{
+		Name:  DBSyncV3,
+		Usage: "[Optional] Use Xray DBSync V3",
+	},
+	PeriodicDBSyncV3: cli.BoolFlag{
+		Name:  PeriodicDBSyncV3,
+		Usage: fmt.Sprintf("[Optional] Get Xray DBSync V3 Periodic Package (Use with %s flag)", DBSyncV3),
```
Comment (eyalbe4) [inline utils/cliutils/commandsflags.go:1162]: ...(Use with %s flag)... --> ...(Use with the --%s command option)...

Also, why is it necessary to have these two separate options? Is `DBSyncV3` sometimes used without `PeriodicDBSyncV3`?

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: utils/cliutils/commandsflags.go (hunk 1 above)
Comment (itraviv) [reply utils/cliutils/commandsflags.go:1162]: The DBSync V3 Feature has 2 states:

```--dbsyncv3 ```

Will download the full on-boarding package containing all the data (as same as happening now, when you omit the ```--from``` and ```--to``` flags)

and

```--dbsyncv3 --periodic ```

Will download a periodic update package. This is replacing the ```--from``` and ```--to``` flags.

So to summarize:
when full on-boarding  we will use ```--dbsyncv3``` 
when doing regular updates we will use ```--dbsyncv3 --periodic```

File: go.mod [hunk 2]
```diff
@@ -95,4 +95,6 @@ replace github.com/jfrog/jfrog-client-go => github.com/jfrog/jfrog-client-go v1.
 
 replace github.com/jfrog/build-info-go => github.com/jfrog/build-info-go v1.3.0
 
-replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.15.3-0.20220607100216-b587c75985e2
+//replace github.com/jfrog/jfrog-cli-core/v2 => github.com/jfrog/jfrog-cli-core/v2 v2.15.3-0.20220607100216-b587c75985e2
```
[the remark this answers is not in the document]
Comment (eyalbe4) [reply go.mod:98]: @itraviv, now that the jfrog-cli-core part of this feature is merged, you can update this `replace` to point to `dev` and we'll merge this PR.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]

File: go.mod (hunk 2 above)
[the remark this answers is not in the document]
Comment (itraviv) [reply go.mod]: yea sure, just going over what you have merged i noticed i missed an error checking. please let's merge [this](https://github.com/jfrog/jfrog-cli-core/pull/428), and i will move forward with this also

[the remark this answers is not in the document]
Comment (eyalbe4) [reply go.mod]: Done @itraviv

**Pull request merged**